CONTRIBUTING.md

```diff
@@ -72,28 +72,58 @@ If you need to override SemVer behavior (not recommended):
 
 The last line of the script's log will give you the command you need to execute to push the commit and tag.
 
-### Release a major version (example)
+### Release latest version (example)
```

Comment:
Only suggestion:

```suggestion
### Release the latest version
```

Reply:
BTW, removing "example" here from title as these seem to be regular instructions, as where 2nd one is an example.

Reply:
Probably native speaker could answer better, but my thinking is that "latest" is only one so it's `the` 🤔